### ISSUE TITLE
Lexbor seems to be the preferred choice?

### ISSUE THREAD

**Issue by benjaoming** [rushter/selectolax#82]
Thanks for a great project!

I saw this in the README:

> Selectolax supports two backends: Modest and Lexbor. By default, all examples use the Modest backend. Most of the features between backends are almost identical, but there are still some differences.
>
> Currently, the Lexbor backend is in beta and missing some of the features.

It seems that development on Modest was stopped some years ago and the Modest README also says to use Lexbor.

Lexbor is version 3.0.4.

**Comment by rushter** [comment]
The development of lexbor have resumed recently. Personally, I still use Modest.

**Comment by benjaoming** [comment]
@rushter that's fair! If you don't feel like it's valuable to change it right now, then feel free to close the issue :+1: 

Can also leave it open in case others have experience with how the two libraries currently compare and if the "beta" label makes sense.

**Comment by lexborisov** [comment]
@rushter  @benjaoming 

I don't quite understand so much love for the Modest project. Maybe the name is good :)

I wouldn't use Modest at all.
Lexbor is much better, both architecturally and in terms of speed and resource consumption.

Also Lexbor is much better developed in terms of CSS. It supports CSSOM stylesheets, applying styles to html elements, and so on.

But here everyone decides for himself what he chooses.

**Comment by rushter** [comment]
> I don't quite understand so much love for the Modest project. Maybe the name is good :)

Some of the features were missing. I need to check what's missing. Modest backend supports a little bit more features.

**Comment by lexborisov** [comment]
@rushter 

I would be happy to explore and implement all the missing components.
If you provide such a list, that would be great!

**Comment by rushter** [comment]
> Hi is there any progress on this issue?
> 
> Last update to modest was almost 2 years ago
> 
> And there seems to be a new version of lexbor

Can you please elaborate on this? What kind of issues do you have?

I can't replace Modest with lexbor because they behave differently.
There will be two backends in the near future.
I usually update lexbor when it gets major updates. It was update 3 months ago.

**Comment by baseplate-admin** [comment]
Sorry, I did not mean to offend you in any way.

> I can't replace Modest with lexbor because they behave differently.

I was going to argue with you about this point. 

>  There will be two backends in the near future.

But if you feel like having 2 backends is a good enough for `selectolax`. fair enough i wont complain.

> I usually update lexbor when it gets major updates.

It seemed like lexbor [updated](https://github.com/lexbor/lexbor/commit/039b432aaf625f29078a5ec28df6c8941e31199e) their [engine](https://github.com/lexbor/lexbor/commit/b220ea3b54b92dbadb85059b22c0a7725fe902e8) with some improvements.

**Comment by benjaoming** [comment]
@baseplate-admin yes, definitely the point that @rushter makes is that `selectolax` _also_ supports Lexbor.. so both Lexbor and Modest are supported, and this issue wasn't contending to remove support for Modest.

Using Modest can be a fine choice for some projects, but I'd argue that the new default should be Lexbor. But that shouldn't replace Modest, it can still be around for projects that need it.

There's also some text in the README that should be updated.
